features/bevragen/zoek-met-nummeraanduiding-identificatie/dev/parameter-autorisatie-gba.feature

```diff
@@ -63,26 +58,60 @@ Functionaliteit: autorisatie op parameters bij ZoekMetNummeraanduidingIdentifica
       | code     | unauthorizedParameter                                                                 |
       | instance | /haalcentraal/api/brp/personen                                                        |
 
+    Scenario: Afnemer zoekt met alleen nummeraanduidingIdentificatie en heeft uitsluitend de autorisatie die nodig is om deze vraag te mogen stellen
+      Gegeven de afnemer met indicatie '000008' heeft de volgende 'autorisatie' gegevens
+      | Rubrieknummer ad hoc (35.95.60) | Medium ad hoc (35.95.67) | Datum ingang (35.99.98) |
+      | 10120 81190                     | N                        | 20201128                |
+      En de geauthenticeerde consumer heeft de volgende 'claim' gegevens
+      | naam         | waarde |
+      | afnemerID    | 000008 |
+      | gemeenteCode | 0518   |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                          | waarde                               |
+      | type                          | ZoekMetNummeraanduidingIdentificatie |
+      | nummeraanduidingIdentificatie | 0599200000219679                     |
+      | gemeenteVanInschrijving       | 0599                                 |
```

Comment:
Jouw scenario geeft aan alleen te zoeken op num ident en je hebt alleen rechten daarop. gemeenteVanInschrijving gebruik je anders ook en dus dien je hier een foutmelding op te krijgen met code unauthorizedParameter.

Is er een rule dat ik mis?

Reply:
hier hoort de parameter gemeenteVanInschrijving niet tussen. Die zit juist in het volgende scenario